src/external/wannier.jl

```diff
@@ -0,0 +1,36 @@
+"""
+Use Wannier.jl to read a .win file and produce the nnkp data (i.e. the b-vectors used in the Mmn matrix).
+"""
+function get_nnkpt_from_wannier(fileprefix)
+    model = Wannier.read_w90(fileprefix; amn=false, mmn=false, eig=false)
+    bvectors = model.bvectors
+
+    nnkpts = reduce(vcat,
+        map(1:model.n_kpts) do ik
+            zp = zip(bvectors.kpb_k[:, ik], eachcol(bvectors.kpb_b[:, :, ik]))
+            map(zp) do (ikpb, G_shift)
+                (ik, ikpb, G_shift=copy(G_shift))
+            end
+        end)
+
+    (nntot=model.n_bvecs, nnkpts)
+end
+
+"""
+Build a Wannier.jl model that can be used for Wannierization.
+"""
+@timing function get_wannier_model(scfres;
```

Comment:
I think I would drop the `get_` and just call it `wannier_model` or if the type in Wannier.jl is always the same and Wannier.jl uses the convention of having constructors that "do things (instead of just storing data), it can even be an additional constructor method (e.g. if the type is `WannierModel` define a method `Wannier.WannierModel( ... )` with the arguments you pass.

Reply:
I went for `wannier_model` because the type is unfortunately `Wannier.Model` and that might conflict with DFTK's own `Model` type.

Reply:
Ah yes, that's probably true. In that case `wannier_model` is probably a good name.